File: python/coupon/does_coupon_code_exist.py
from ultracart.apis import CouponApi
from samples import api_client

coupon_api = CouponApi(api_client())
merchant_code = '10OFF'

api_response = coupon_api.does_coupon_code_exist(merchant_code)
coupon_exists = api_response.exists

print(api_response)

File: ruby/coupon/does_coupon_code_exist.rb
require_relative '../constants'
require 'ultracart_api'

coupon_api = UltracartClient::CouponApi.new_using_api_key(Constants::API_KEY)
merchant_code = '10OFF'

api_response = coupon_api.does_coupon_code_exist(merchant_code)
coupon_exists = api_response.exists

puts api_response.inspect